dev-docs/modules/userid-submodules/33across.md

```diff
@@ -30,6 +30,7 @@ The following configuration parameters are available:
 | name | Required | String | The name of this sub-module | `"33acrossId"` |
 | params ||| Details for the sub-module initialization ||
 | params.pid | Required | String | Partner ID (PID) | Please reach out to [PrebidUIM@33across.com](mailto:PrebidUIM@33across.com) and request your PID |
+| params.hem | Optional | String | Hashed email address in sha256 format | `"ba4235544d6c91865fb07.."` |
```

Comment:
this looks fine, but might consider adding the Prebid.js version where this parameter is supported once you know.